lib/Runtime/ByteCode/OpCodes.h

```diff
@@ -511,8 +511,8 @@ MACRO_EXTEND_WMS_AND_PROFILED(NewScObjectSpread,   CallIExtended, OpSideEffect|O
 MACRO_WMS_PROFILED2(    NewScObjArray,      CallI,          OpSideEffect|OpUseAllFields|OpCallInstr)        // Create new ScriptObject instance
 MACRO_WMS_PROFILED2(    NewScObjArraySpread, CallIExtended, OpSideEffect|OpUseAllFields|OpCallInstr)        // Create new ScriptObject instance
 MACRO(                  NewScObject_A,      Auxiliary,      OpSideEffect|OpUseAllFields)                    // Create new ScriptObject instance passing only constants
-MACRO_WMS(              NewScObjectNoCtorFull, Reg2,        OpHasImplicitCall | OpTempObjectCanStoreTemp)   // Create new object that will be used for the 'this' binding in a base class constructor
-MACRO_BACKEND_ONLY(     NewScObjectNoCtor,  Empty,          OpHasImplicitCall | OpTempObjectCanStoreTemp)   // Create new object that will be passed into a constructor
+    MACRO_WMS(              NewScObjectNoCtorFull, Reg2,        /*OpHasImplicitCall |*/ OpTempObjectCanStoreTemp)   // Create new object that will be used for the 'this' binding in a base class constructor
```

Comment:
Thanks. Thought I’d caught all of these.

--Paul

From: Curtis Man [mailto:notifications@github.com]
Sent: Thursday, June 23, 2016 10:18 AM
To: Microsoft/ChakraCore <ChakraCore@noreply.github.com>
Cc: Paul Leathers <pleath@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Microsoft/ChakraCore] Restructure OpHasImplicitCall/OpCallsValueOf (#1162)


In lib/Runtime/ByteCode/OpCodes.h<https://github.com/Microsoft/ChakraCore/pull/1162#discussion_r68276001>:

> @@ -511,8 +511,8 @@ MACRO_EXTEND_WMS_AND_PROFILED(NewScObjectSpread,   CallIExtended, OpSideEffect|O

>  MACRO_WMS_PROFILED2(    NewScObjArray,      CallI,          OpSideEffect|OpUseAllFields|OpCallInstr)        // Create new ScriptObject instance

>  MACRO_WMS_PROFILED2(    NewScObjArraySpread, CallIExtended, OpSideEffect|OpUseAllFields|OpCallInstr)        // Create new ScriptObject instance

>  MACRO(                  NewScObject_A,      Auxiliary,      OpSideEffect|OpUseAllFields)                    // Create new ScriptObject instance passing only constants

> -MACRO_WMS(              NewScObjectNoCtorFull, Reg2,        OpHasImplicitCall | OpTempObjectCanStoreTemp)   // Create new object that will be used for the 'this' binding in a base class constructor

> -MACRO_BACKEND_ONLY(     NewScObjectNoCtor,  Empty,          OpHasImplicitCall | OpTempObjectCanStoreTemp)   // Create new object that will be passed into a constructor

> +    MACRO_WMS(              NewScObjectNoCtorFull, Reg2,        /*OpHasImplicitCall |*/ OpTempObjectCanStoreTemp)   // Create new object that will be used for the 'this' binding in a base class constructor

Remove commented code?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/ChakraCore/pull/1162/files/399fcf51c410a7ab619374ffc00f5b0ba4bbf54d#r68276001>, or mute the thread<https://github.com/notifications/unsubscribe/APF8RGqhnFHbpA37dzkn3rHKOtupCD7Pks5qOr_VgaJpZM4I7C5E>.